rebar.config

```diff
@@ -45,9 +45,6 @@
             {config, "config/test.config"}
         ]}
     ]},
-    {docs, [
-        {erl_opts, [{d, 'DOC'}]}
```

Comment:
Nope, you misread, I said the `doc` ALIAS, line 35
```
{alias, [
    {analyze, [xref, dialyzer]},
    {doc, [{do, "default as doc ex_doc"}]}, <---- REMOVE
    {analyze, [xref, dialyzer]}
]}.
```
The profile is still needed, to apply the define. We just do not need the alias to use it.

Reply:
Do not know if you can see the CI run errors https://github.com/grisp/grisp/actions/runs/12817450554/job/35740695885
No alias, Yes profile

Reply:
When you build the documentation locally, always delete _build and artifacts under doc/ because otherwise it lloks like it is still working even though you broke something.

Reply:
Ah yes! Fixed it.